.coveragerc

```diff
@@ -2,6 +2,7 @@
 branch = true
 source =
   xonsh/
+  xontrib/
```

Comment:
@astronouth7303 was `xontrib/` omitted from coverage intentionally, or was it just an oversight?

Reply:
tbh, It's been too long, I have no idea.

It probably has to do with the testability of a lot of the xontribs?